app/extensions.js

```diff
@@ -294,6 +298,47 @@ let generateTorrentManifest = () => {
   }
 }
 
+let generateEthwalletManifest = () => {
+  let cspDirectives = {
+    'default-src': '\'self\'',
+    'style-src': '\'self\' \'unsafe-inline\'',
+    'connect-src': 'blob: \'self\' ws://localhost:* http://localhost:* https://min-api.cryptocompare.com https://mini-api.cryptocompare.com',
+    'img-src': '\'self\' data:',
+    'script-src': '\'self\' \'unsafe-eval\' \'sha256-7B6rTuXUsu9shBeECmDFH4h7RDsfogQ3kIonJnIL40o\' \'sha256-zgjB35Pd2ax7Wwfk9iKnAH8r+gNrD2cHpxDkH81DHzw=\'',
```

Comment:
@Slava do we need unsafe-eval here?

Reply:
I think I added it because whatever the Meteor app compiles into uses `new Function`. I can look into how tweaking the minifier and try to get something that doesn't use eval.

Reply:
Ok after spending some time on this I tracked it down to this 
https://github.com/facebook/regenerator/issues/336. It is an unsafe line in regenerator that used to compile web3js package for Meteor.

Reply:
hmm maybe not exactly this, but something very similar that tries to get the value of global